core/dev-tools/templates/simple/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<title>Luigi Simple Dev</title>
+	<link rel='stylesheet' href='./public/luigi.css'>
+  <meta charset="utf-8">
+</head>
+
+<body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script src="./public/luigi.js"></script>
+	  <script src="luigi-config.js"></script>

```

Comment:
Relative paths are not working as soon as you navigate to a deeper path.
```suggestion
	<link rel='stylesheet' href='/public/luigi.css'>
  <meta charset="utf-8">
</head>

<body>
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <script src="/public/luigi.js"></script>
	  <script src="/luigi-config.js"></script>

```